Sources/BraintreePayPalMessaging/BTPayPalMessagingRequest.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+/// Used to initialize a `BTPayPalMessagingClient`
+/// This feature is currently only supported for buyers located in the US. For merchants domiciled outside of the US
+/// please set the `buyerCountry` to display messaging to US based buyers.
+/// - Note: This module is in beta. It's public API may change or be removed in future releases.
+public struct BTPayPalMessagingRequest {
+
+    var amount: Double?
+    var placement: BTPayPalMessagingPlacement?
+    var offerType: BTPayPalMessagingOfferType?
+    var buyerCountry: String?
+    var logoType: BTPayPalMessagingLogoType
+    var textAlignment: BTPayPalMessagingTextAlignment
+    var color: BTPayPalMessagingColor
+    
+    /// Initialize a `BTPayPalMessaging`
+    /// - Parameters:
+    ///   - amount: Price expressed in cents amount based on the current context (i.e. individual product price vs total cart price)
+    ///   - placement: Message screen location (e.g. product, cart, home)
+    ///   - offerType: Preferred message offer to display
+    ///   - buyerCountry: Consumer's country (Integrations must be approved by PayPal to use this option)
+    ///   - logoType: Logo type option for a PayPal Message. Defaults to `.inline`
+    ///   - textAlignment: Text alignment option for a PayPal Message. Defaults to `.right`
+    ///   - color: Text and logo color option for a PayPal Message. Defaults to `.black`
+    public init(
+        amount: Double? = nil,
+        placement: BTPayPalMessagingPlacement? = nil,
+        offerType: BTPayPalMessagingOfferType? = nil,
+        buyerCountry: String? = nil,
+        logoType: BTPayPalMessagingLogoType = .inline,
+        textAlignment: BTPayPalMessagingTextAlignment = .right,
```

Comment:
Should the default be left aligned?

Reply:
I based these on the defaults in the `PayPalMessages`, but not entirely sure why they went right instead of left. Technically we can do whatever we want, so if we prefer left we can do that.

Reply:
If that's the default in PayPalMessages it's fine with me 🤷

Reply:
let's get confirmation from PP friends which is preferred default. I want to make sure we have parity across PP and BT

Reply:
`right` is their default, see: https://github.com/paypal/paypal-messages-ios/blob/6904fdcc27b51c3dc0e05356423b618d584a925b/Sources/PayPalMessages/Config/PayPalMessageConfig.swift#L77